ZuluContent/Items/Jewels/BaseJewel.cs

```diff
@@ -98,6 +105,13 @@ public CraftResource Resource
             }
         }
 
+        [CommandProperty(AccessLevel.GameMaster)]
+        public MarkQuality Mark
+        {
+            get => Enchantments.Get((ItemMark e) => (MarkQuality) e.Value);
```

Comment:
Even though this supports enchantments it's probably best to keep it consistent and use the `ICraftable` serialization for `MarkQuality` here